### PR TITLE
Sara

### PR DESCRIPTION
**Pull request by saranatour1**
Made the orders, pending and delivered ,  must add a status  check to make the orders more appropriate, the code is full of bugs 😃. 
the changes will be discussed.

### CODE REVIEW

**Comment by saranatour1** [comment]
😄

File: Feedme/src/main/java/com/codingdojo/feedme/services/OrderServices.java
Comment (Yousef-labadi) [inline Feedme/src/main/java/com/codingdojo/feedme/services/OrderServices.java:1]: public int countPendingOrders(Long id)  delete this method

**Review by Yousef-labadi — APPROVED** [review]
reviwed all changes , make sure to update base all comment

File: Feedme/src/main/java/com/codingdojo/feedme/services/OrderServices.java
Comment (saranatour1) [reply Feedme/src/main/java/com/codingdojo/feedme/services/OrderServices.java:1]: we will make sure to update everything accordingly.

**Pull request merged**